h/services/annotation_json_presentation/service.py

```diff
@@ -2,26 +2,29 @@
 
 from h import storage
 from h.models import Annotation
-from h.presenters import AnnotationJSONPresenter
 from h.security import Identity, identity_permits
 from h.security.permissions import Permission
+from h.services.annotation_json_presentation._basic_presenter import BasicJSONPresenter
 from h.traversal import AnnotationContext
 
 
 class AnnotationJSONPresentationService:
     def __init__(  # pylint: disable=too-many-arguments
         self, session, links_svc, flag_svc, user_svc
     ):
-        self.session = session
-        self.links_svc = links_svc
-        self.flag_svc = flag_svc
-        self.user_svc = user_svc
-        self._presenter = AnnotationJSONPresenter(
-            links_service=self.links_svc, user_service=self.user_svc
+        self._session = session
+        self._links_service = links_svc
+        self._user_service = user_svc
+        self._flag_service = flag_svc
+        self._presenter = BasicJSONPresenter(
+            links_service=self._links_service, user_service=self._user_service
         )
 
+    def present(self, annotation):
+        return self._presenter.present(annotation)
```

Comment:
`present_for_user` could be `present` with an optional user param and a conditional. But more explicitly named with two methods works as well :+1: